src/init.ts

```diff
@@ -0,0 +1,85 @@
+import { JaaS_DOMAIN } from "./constants";
+
+type ExternalApiMap = {
+  [key: string]: {
+    fn?: any;
+    isLoaded: boolean;
+    callbacks: ((err: Error | null, jitsiClass?: any) => void)[];
+    err: Error | null;
+  };
+};
+
+const externalApiMap: ExternalApiMap = {};
+
+/**
+ * Injects the external_api.js script for the corresponding domain in DOM
+ * and resolves with an object containing either the `JitsiMeetExternalAPI`
+ * class definition or an error
+ *
+ * @param {string} domain - The domain of the external API
+ * @returns {Promise<object>} - Object containing the JitsiMeetExternalAPI object
+ * or an error
+ */
+export const initExternal = (domain: string = JaaS_DOMAIN): Promise<void> =>
+  new Promise((resolve, reject) => {
+    init(domain, (error: Error | null, jitsiClass?: any): void => {
+      if (error) {
+        reject(error);
+      }
+      resolve(jitsiClass);
+    });
+  });
+
+export const init = (
```

Comment:
Please give this a more descriptive name such as `initExternalAPI` and make the default domain be `meet.jit.si`.